### DISCUSSION TITLE
Duvida

### DISCUSSION THREAD

**Discussion by Fernando1857** [Zian25/UniTools-Termux#7] [Q&A]
Pra q serve? Instalei o facebrut, o q faço pra hackear agora?

**Comment by Ahosall** [comment]
> Pra q serve? Instalei o facebrut, o q faço pra hackear agora?

Recomendo você estudar um pouco sobre bforce .... se essa ferramenta for um painel você terá que acessar a pasta dele e executar o arquivo principal, agora se for um comando execute o comando acompanhado de `--help`>

Boa sorte 😉
